plugins/newsdownloader.koplugin/main.lua

```diff
@@ -5,11 +5,19 @@ local DataStorage = require("datastorage")
 local DownloadBackend = require("epubdownloadbackend")
 local ReadHistory = require("readhistory")
 local FFIUtil = require("ffi/util")
+local FeedView = require("feed_view")
 local InfoMessage = require("ui/widget/infomessage")
 local InputDialog = require("ui/widget/inputdialog")
 local LuaSettings = require("frontend/luasettings")
 local UIManager = require("ui/uimanager")
+<<<<<<< HEAD
+=======
+local KeyValuePage = require("ui/widget/keyvaluepage")
+local InputDialog = require("ui/widget/inputdialog")
+local MultiConfirmBox = require("ui/widget/multiconfirmbox")
+>>>>>>> feed_view
```

Comment:
Lots of merge conflicts remnants.

Reply:
Great! I can't believe I did that again! >:)